examples/openai/agent_with_pre_planned_tools.py

```
@@ -24,53 +24,96 @@ def main() -> None:
     rprint(brave_search_function_definition)
 
-    response = openai.chat.completions.create(
-        model="gpt-4o",
-        messages=[
+    github_star_repository_function_definition = aci.functions.get_definition(
+        "GITHUB__STAR_REPOSITORY"
+    )
+    rprint(Panel("Github star repository function definition", style="bold blue"))
+    rprint(github_star_repository_function_definition)
+
+    messages=[
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)